app/src/main/res/layout/activity_main.xml

```diff
@@ -49,7 +49,8 @@
             android:layout_marginStart="64dp"
             android:layout_marginTop="28dp"
             android:layout_marginEnd="64dp"
-            android:text="Lorem ipsum dolor sit amet"
+            android:text="@string/main_activity_solgan"
```

Comment:
You're including changes here for your other task. You'll either have to remove these or wait until you've merged this PR before this one can be merged.